app/src/components/map.jsx

```diff
@@ -39,19 +39,19 @@ class Map extends Component {
 
     this.setState({running: !!!this.state.running});
     requestAnimationFrame(function() {
-      this.animateMapData(this.state.ite || min2015 , mDay);
+      this.animateMapData(this.state.ite || tmlnMinDate , mDay);
     }.bind(this));
   }
 
   animateMapData(ite) {
     if (!this.state.running) return;
 
     ite = ite || 0;
-    if (ite > max2015) {
+    if (ite > tmlnMaxDate) {
       this.setState({running: !!!this.state.running, ite: null, widthRange: 0});
       return ;
     }
-    let width = ((ite - min2015) / mDay)/ ((max2015 - min2015) / mDay) * 100;
+    let width = ((ite - tmlnMinDate) / mDay)/ ((tmlnMaxDate - tmlnMinDate) / mDay) * 100;
```

Comment:
not for now, but in the future, please use more human-friendly variable names. minifiers can shorten them later on, and when you start operating over them like this, it's easy to lose track of what's really going on.